finance/bitcoin/Portfile

```diff
@@ -22,23 +22,31 @@ master_sites            ${homepage}bin/bitcoin-core-${version}/
 checksums           rmd160  a11078445daad75bbe2032da307d292670eece55 \
                     sha256  34de2dbe058c1f8b6464494468ebe2ff0422614203d292da1c6458d6f87342b4
 
-depends_build           port:pkgconfig \
-                        port:autoconf \
-                        port:automake \
-                        port:libtool
-
-depends_lib             port:boost \
-                        path:lib/libssl.dylib:openssl \
-                        port:db48 \
-                        port:libevent \
-                        port:miniupnpc \
-                        port:protobuf-cpp \
-                        port:zmq
+depends_build	port:autoconf port:automake port:libtool port:pkgconfig
+depends_lib	port:boost port:libevent port:miniupnpc port:openssl \
+		port:protobuf-cpp port:python36 port:qt5 port:zmq
 
 use_parallel_build      yes
 
-configure.args          --disable-ccache \
-                        --disable-silent-rules
+configure.args          --disable-ccache		\
+			--disable-silent-rules		\
+			--disable-upnp-default		\
+			--enable-tests			\
+			--disable-bench			\
+			--disable-hardening		\
+			--disable-reduce-exports	\
+			--disable-lcov			\
+			--disable-glibc-back-compat	\
+			--disable-experimental-asm	\
+			--enable-zmq			\
+			--enable-man			\
+			--disable-debug			\
+			--enable-werror			\
+			--enable-largefile		\
+			--with-miniupnpc		\
+			--without-system-univalue	\
+			--with-utils			\
+			--with-libs			\
```

Comment:
You're using tabs for indentation here. Please use spaces, as mandated by the modeline.